package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "https-proxy-agent",
-  "version": "2.2.2",
+  "version": "2.2.3",
```

Comment:
You should probably leave this to the maintainers.

Reply:
ok